vcdh5cc.l10n

```diff
@@ -0,0 +1,27 @@
+#
```

Comment:
Please remove existing comments and add a few lines of our own saying that these translations are maintained by VMware internally and this file contains all the locations in this repo that must be localized

Reply:
Updated the main comment at the top